src/worker.cpp

```diff
@@ -417,6 +417,8 @@ void Worker::setDeviceTreeAndJson()
                     ec.message());
             }
         }
+
+        m_isFactoryResetDone = true;
```

Comment:
I think "m_isFactoryResetApplied" suits more

Reply:
No, Applied as a different meaning.
Done implies that an action has been done. This variable denotes that when we are reaching this point of code, there has been a factory reset done in the system.